app/app/components/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styles from "./Button.module.css";
+
+const Button = ({ onClick, children, type = "button", variant = "primary", className }) => {
```

Comment:
Instead of creating your own components here, I would suggest using a pre made component library. This will save a lot of time.

Just one example: https://mui.com/